grr/server/grr_response_server/bin/grrafana.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+# Lint as: python3
+import os
+import json
+import typing
```

Comment:
Instead of importing `typing` and using `typing.cast`, please add `cast` to the line below. I.e.:
`from typing import cast, Any, Dict, List, Text, Tuple`

Reply:
I made note and changed everything to do with the imports; please let me know if it's OK!